### PR TITLE
Add vulkan image and show size

### PR DESCRIPTION
**Pull request by ericcurtin**
Add this image, we also want to see the size of this image in the build.

Co-authored-by: Steffen Roecker <sroecker@redhat.com>
Signed-off-by: Eric Curtin <ecurtin@redhat.com>

### CODE REVIEW

**Comment by ericcurtin** [comment]
This is a replacement for https://github.com/containers/ramalama/pull/270

File: container-images/vulkan/Containerfile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM quay.io/ramalama/ramalama:latest
+
+RUN /usr/bin/python3 --version
+
+RUN git clone https://github.com/ggerganov/llama.cpp && \
+    cd llama.cpp && \
+    git reset --hard ${LLAMA_CPP_SHA} && \
```
Comment (ericcurtin) [inline container-images/vulkan/Containerfile:7]: @rhatdan do you know if ARGS like LLAMA_CPP_SHA and WHISPER_CPP_SHA get inherited from parent container images?

Comment (ericcurtin) [reply container-images/vulkan/Containerfile:11]: I'm assuming they don't, but I don't know

Comment (rhatdan) [reply container-images/vulkan/Containerfile:11]: Only if they are set as Environment variables, which currently they are not.  If we want this to be set we could do it via environment in ramalama Containerfile

env LLAMA_CPP_SHA=${LLAMA_CPP_SHA}

Comment (rhatdan) [reply container-images/vulkan/Containerfile:11]: Trying it out now.

Comment (rhatdan) [reply container-images/vulkan/Containerfile:11]: Works: https://github.com/containers/ramalama/pull/355

Comment (ericcurtin) [reply container-images/vulkan/Containerfile:11]: Cool that's better, I'll merge yours

File: container-images/rocm/Containerfile [hunk 2]
```diff
@@ -2,6 +2,9 @@ FROM quay.io/ramalama/ramalama:latest
 
 RUN /usr/bin/python3 --version
 
+ARG LLAMA_CPP_SHA=3f1ae2e32cde00c39b96be6d01c2997c29bae555
```
Comment (rhatdan) [inline container-images/rocm/Containerfile:5]: Remove lines.

File: container-images/vulkan/Containerfile [hunk 3]
```diff
@@ -0,0 +1,27 @@
+FROM quay.io/ramalama/ramalama:latest
+
+ARG LLAMA_CPP_SHA=3f1ae2e32cde00c39b96be6d01c2997c29bae555
```
Comment (rhatdan) [inline container-images/vulkan/Containerfile:3]: Remove Lines

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**